drivers/SmartThings/matter-media/src/init.lua

```diff
@@ -15,7 +15,7 @@
 local capabilities = require "st.capabilities"
 local clusters = require "st.matter.clusters"
 local MatterDriver = require "st.matter.driver"
-local utils = require "st.utils"
+local MediaPlaybackClusterAcceptedCommandList = clusters.MediaPlayback.attributes.AcceptedCommandList
```

Comment:
I think this should be somewhat safe to merge at this point since 0.46.x has been distributed all the way to external beta populations and they are the only ones that would really be running the driver. But this will cause errors on any hub running 0.45.x because the attribute does not exist yet.